forge.config.ts

```
@@ -12,6 +12,17 @@ const config: ForgeConfig = {
     asar: true,
     extraResource: [".vite/build/ffmpeg", "node_modules/mulmocast/assets", "node_modules/mulmocast/scripts"],
     icon: "./images/macoro.png",
+    osxSign: {
+      identity: process.env.CODESIGN_IDENTITY!,
+      hardenedRuntime: true,
+      entitlements: "entitlements.plist",
+      entitlementsInherit: "entitlements.plist",
+    } as any,
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)